README.md

```diff
@@ -0,0 +1,286 @@
+
+### If installing for deployment, run:
+```
+pip install arches-modular-reports
+```
+
+
+## Project Configuration
+
+1. If you don't already have an Arches project, you'll need to create one by following the instructions in the Arches [documentation](http://archesproject.org/documentation/).
+
+2. When your project is ready add "arches_modular_reports", "arches_querysets", and "arches_component_lab" to INSTALLED_APPS **below** the name of your project:
+    ```
+    INSTALLED_APPS = (
+        ...
+        "my_project_name",
+        "arches_modular_reports",
+        "arches_querysets",
+        "arches_component_lab"
+    )
+    ```
+
+3. Next ensure arches, arches_modular_reports, dayjs, @primevue/forms, and @primeuix/themes are included as dependencies in package.json
+    ```
+    "dependencies": {
+        "arches": "archesproject/arches#stable/7.6.12",
+        "arches-modular-reports": "archesproject/arches-modular-reports#main",
+        "dayjs": "^1.11.13",
+        "@primevue/forms": "4.3.3",
+        "@primeuix/themes": "1.0.1"
+    }
+    ```
+
+4. Update urls.py to include the arches-modular-reports urls
+    ```
+    urlpatterns = [
+        ...
+    ]
+
+    urlpatterns.append(path("", include("arches_modular_reports.urls")))
+
+    # Ensure Arches core urls are superseded by project-level urls
+    urlpatterns.append(path("", include("arches.urls")))
+
+    ```
+
+5. Run migrations
+    ```
+    python manage.py migrate
+    ```
+
+6. Start your project
+    ```
+    python manage.py runserver
+    ```
+
+7. Next cd into your project's app directory (the one with package.json) install and build front-end dependencies:
+    ```
+    npm install
+    npm run build_development
+    ```
+
+## Setting up a graph to use the Modular Reports Template
+
+Once you've installed the Arches Modular Report Application into your project you'll notice a new report template available called "Modular Report Template".  
+
+1. Select a Graph in the graph designer that you'd like to use with the new modular reports.
+
+2. Navigate to the "Cards" tab, select the root node and select the "Modular Report Template" from the Report Configuration section on the right.
+
+3. Next go to the [admin page](https://arches.readthedocs.io/en/stable/administering/django-admin-ui/) and login.
+
+4. Click on the "+ Add" button next to the item called "Report configs" under the "Arches Modular Reports" section.
+ 
+5. You'll be presented with a large "Config" section that should only comtain empty curly brackets "{}".  Below that is a dropdown with a listing of graphs available in your project.  Select the graph you chose earlier in step 1 and then click the button that says "Save and continue editing".
```

Comment:
I could cook up some quick on-screen help text for this.

Reply:
setting `help_text` on the model field gives this, is it useful?

<img width="883" alt="Screenshot 2025-06-27 at 12 20 54 PM" src="https://github.com/user-attachments/assets/ad600037-5a69-4a6a-a7c5-7c4046467192" />